src/main/java/io/hops/hopsworks/expat/migrations/docker/JobsDockerCommandArgsMigration.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * This file is part of Expat
+ * Copyright (C) 2021, Logical Clocks AB. All rights reserved
+ *
+ * Expat is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU Affero General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * Expat is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with
+ * this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package io.hops.hopsworks.expat.migrations.docker;
+
+import io.hops.hopsworks.expat.db.DbConnectionFactory;
+import io.hops.hopsworks.expat.migrations.MigrateStep;
+import io.hops.hopsworks.expat.migrations.MigrationException;
+import io.hops.hopsworks.expat.migrations.RollbackException;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.JSONObject;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class JobsDockerCommandArgsMigration implements MigrateStep {
+
+  private static final Logger LOGGER = LogManager.getLogger(JobsDockerResourcesMigration.class);
```

Comment:
Change logger to JobsDockerCommandArgsMigration

Reply:
fixed